client/my-sites/earn/ads/form-settings.jsx

```diff
@@ -342,32 +342,21 @@ class AdsFormSettings extends Component {
 				<FormFieldset>
 					<SupportInfo
 						text={ translate(
-							'Enables a targeted advertising opt-out link for California consumers, as required by the California Consumer Privacy Act (CCPA).'
+							'Enables a targeted advertising opt-out link in US States where this is legally required.'
```

Comment:
Minor: should the "S" in "US states" be capitalized? For example, it isn't capitalized on line 353 below.

```suggestion
							'Enables a targeted advertising opt-out link in US states where this is legally required.'
```